lib/logstash/inputs/gelf.rb

```diff
@@ -68,36 +76,129 @@ def initialize(params)
   public
   def register
     require 'gelfd'
+    @tcp = nil
+    @udp = nil
+    if @port_tcp == nil
+      @port_tcp = @port
+    end
+    if @port_udp == nil
+      @port_udp = @port
+    end  
   end # def register
 
   public
   def run(output_queue)
     begin
-      # udp server
-      udp_listener(output_queue)
+      if @use_tcp
+        tcp_thr = Thread.new(output_queue) do |output_queue|
+          tcp_listener(output_queue)
+        end
+      end
+      if @use_udp
+        udp_thr = Thread.new(output_queue) do |output_queue|
+          udp_listener(output_queue)
```

Comment:
This looks like it is braking the tests.

Reply:
I created a branch on my fork to try and fix the test. How can I run the travis tests for that branch? I don't want to mess up my master while experimenting what exactly detiriorated the test result